docs/.ruby-version

```diff
@@ -1 +1 @@
-system
+2.4.1
```

Comment:
Why 2.4.1?

Reply:
No reason in particular; just need a non-system version that's >=2.2.